### PR TITLE
small formatting changes

### PR DESCRIPTION
**Pull request by Eriner**
Small formatting changes to make it (more) PEP8 and readable.

I like the dictionary of colours the way it is, PEP8 be damned. It is very readable this way.

### CODE REVIEW

**Comment by rocky** [comment]
> I like the dictionary of colours the way it is, PEP8 be damned. It is very readable this way.

One of the hallmarks of Python is how it annoys most people in at least one or more ways.

Although there are _some_ concessions (more than I'd like) to PEP8, there is a bit of this code that I have written because I prefer it this way. 

I've never had anyone give me a cogent reason as to why the things I do for a considered reason should be done the PEP8 way. The response is almost always, "Well, that's the way everyone else who is following PEP8 does things." My reponse to that is, okay, create a Python 4 that doesn't allow the things PEP8 requires. For example remove the grammar for "import x,y" and make it only "import x". If you think Python3 adoption was slow, let's see how long adoption of Python4 would take. (Answer: probably never)

So here are the things in the formatting that I prefer:

    import os, sys

I don't understand why it is important to have these as separate imports. I'm willing to stipulate all Python programs should import os and sys, so as far as I'm concerned that entire line is garbage. So I'd rather keep it one line of garbage rather than two lines of garbage. (My computer terminal is limited in the number of lines it shows. And my computer terminal has plenty of wasted space to the right even after showing "import os, sys".

    if out_filename: print(out_filename)

Similar reasoning applies here, what's so important about the two lines.
It adds length to the program and leaves lots of white space to the right of both. And you have to worry about indentation when you add another statement afterwards. (Another Python "feature"

Ruby, Perl, C, C++, Java, Javascrit don't care about whether this is one line or two. (In zsh you could use && which I suppose I could do here in Python). Not only that, but in Ruby and Perl they allow:

    print(out_filename) if out_filename

As for
   ["help", "bg=", "colors=",])

I like comma at the end so to make adding and removing elements. In fact things are more regular this way. So again, I don't see why PEP8 makes such a big deal about this.

As for moving the warnings statement or two spaces before '#' okay. The latter rule again I think kinda stupid, but that's the PEP8 folks can be. But here okay I'll cave to the bean counters.

See https://www.youtube.com/watch?v=wf-BqAjZb8M

That said, if you still disagree and can give cogent reasons for the benefits for the PEP8-inspired changes, then by all means lets discuss!

If you want to change the pull request for those two kinds of changes (or any others I haven't commented on), then I'll happily merge. If this is too much trouble for you, that's okay and I'll probably make those remaining changes anyway 

BTW In commit 3f4ebe7c I added a tox.ini for the flake8 things to ignore. If you want to amend the list, I'm amenable. Let's discuss.

**Comment by Eriner** [comment]
I agree with most of your points. The `import`s being on separate lines is just something out of habit and because it makes my editor stop bothering me. Also with the comma, I understand the rational, and it is reasonable. Not having the comma is simply my preference. 

As for the `if out_filename`, I greatly prefer this on 2 lines. This makes scanning code for conditional statements much easier in my opinion. Almost every python program I've worked with formats it like this, so seeing it all on one line was almost jarring. 

To summarize, the `import`s, and the comma is up to you and is entirely preference. That said, I greatly prefer the `if` to span multiple lines with an indent. Seeing as I really only have strong feelings for the one change, I'm going to close this issue since it is easy for you to change it (if you choose to).

**Comment by rocky** [comment]
How do you feel about:

    out_filename and print(out_filename)

instead?

**Comment by Eriner** [comment]
My preference for the `if` likely also comes from my work in shell scripts. Seeing something like:
```
if (( ${+commands[pigz]} )); then alias gzip='pigz'; fi
```

is much less readable than:
```
if (( ${+commands[pigz]}; then
  alias gzip='pigz'
fi
```

It lets you clearly see what the conditional is, then easily determine what the result is. Indentation for conditionals is my preference in every language. Though it may be more verbose and create more lines, it creates a nice visual break.

The only time I really like two 'commands' on one line is in a situation like this:
```
autoload -Uz colors && colors
```

Situations like that where the commands are directly related and are almost always run in sequence are okay to me.

In response to your proposal, I'd still prefer two lines over:
```
out_filename and print(out_filename)
```

**Comment by rocky** [comment]
Ok. 325ee21b7b99062e19b6a78af5b88759bc1c713c has most of this.

A distraction from the more interesting thing of adding 256-color terminal support and pygments styles.

**Comment by Eriner** [comment]
:+1: Promise to do that Monday or Tuesday. Have commitments starting in 34 minutes, but I promise I'll get to that. I want (need) it myself! :)
